src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/locations/LocationsService.kt

```diff
@@ -17,6 +18,13 @@ class LocationsService(@Qualifier("locationsApiWebClient") private val webClient
       .retrieve()
       .awaitBody()
 
+  suspend fun deleteLocation(id: String) {
+    webClient.delete()
+      .uri("/sync/delete/$id")
```

Comment:
this should always be parameterised otherwise will generate new cache for each one and blow the memory of the server eventually

Reply:
Im not sure what you mean .. you dont mean changing it to 
.uri("/sync/delete/{id}", id)
? which seems to just construct the same uri string